src/pages/events/configure-commerce.md

```diff
@@ -61,24 +61,36 @@ Save your changes. The remaining installation steps vary, depending on your envi
    git add app/etc/config.php
    ```
 
-1. Update your `.magento.app.yaml` file to include the following build hook:
+1. If your version of ece-tools is less than `2002.1.13`, make the following modifications to the `composer.json` file. These modifications install ece-tools from the develop branch.
+
+   * Update the `repositories` section so that it matches the following:
```

Comment:
My linting rules for [the API-Mesh-Gateway](https://github.com/AdobeDocs/graphql-mesh-gateway) ask for 2 spaces after unordered list markers. However, I'm not sure if this is an important distinction.

Reply:
it looks like i have a `.json` file with linting rules in that repo, which has the following: `  "MD030": { "ol_single": 1, "ol_multi": 1, "ul_single": 2, "ul_multi": 2 },`. the `extensibility` repo uses a different linting configuration using aliases and doesn't refer to the `list-marker-space` alias. let me know if i should update the `API Mesh Gateway` repo to this format.